### PR TITLE
Update .trigger-ci

### PR DESCRIPTION
**Pull request by arewm**

### CODE REVIEW

**Review by schmidtd — APPROVED** [review]
/lgtm
/shrug

File: .trigger-ci [hunk 1]
```diff
@@ -1 +1 @@
-trigger-02
+23-08-2021
```
Comment (ray-harris) [inline .trigger-ci:1]: Suggest using `2021-08-23`.

Comment (arewm) [reply .trigger-ci:1]: Yes, that makes sense. I was just defaulting to European time format :)

Comment (ray-harris) [reply .trigger-ci:1]: ISO8601 FTW!

**Pull request merged**